src/darwin/Framework/CHIP/MTRDeviceControllerDelegate.h

```diff
@@ -52,8 +55,22 @@ API_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4))
 /**
  * Notify the delegate when commissioning is completed.
  */
-- (void)controller:(MTRDeviceController *)controller commissioningComplete:(NSError * _Nullable)error;
-
+- (void)controller:(MTRDeviceController *)controller
+    commissioningComplete:(NSError * _Nullable)error MTR_NEWLY_DEPRECATED("Please use commissioningComplete:deviceId");
+/**
+ * Notify the delegate when commissioning is completed.
+ * For getting commissioning controllee's deviceId
```

Comment:
_Nonnull_ is correct.  my mistake.

Reply:
@bzbarsky-apple 
I have a question to clarify in your comments.

Since there is no nodeID in the code before the change, I understood that it can be estimated with the nodeID set in the "commission..." command. If so, isn't nodeID not required when it fails?
  -> I think it is possible to change the nodeID parameter to _Nullable in case of failure. (set to nil on failure), how about setting a valid nodeID only on success?

Reply:
@joonhaengHeo That would work, yes, given that the only reason we have the node id bit here is to communicate the node id the device ended up with... if it didn't end up with one, passing in nil would be reasonable.

Reply:
@bzbarsky-apple 
I added error check when commissioning is completed event.